src/com/jalasoft/search/controller/SearchCriteria.java

```diff
@@ -39,7 +35,7 @@ public String getFileName() {
      * @return path
      */
     public String getPath() {
-        log.info(path);
+        getInstance().getLogger().info("Path: " + path);
```

Comment:
I think if the instance return the logger will not be necessary call the Get logger

Reply:
it is necessary to get a logger instance